app/src/main/java/com/example/vre_app/CreateAccount.kt

```diff
@@ -70,20 +70,23 @@ class CreateAccount : AppCompatActivity() {
                         password.text.clear()
                         confirmPassword.text.clear()
 
+                        //Message from the system that password doesn't match
                     } else if (password.text.toString() != confirmPassword.text.toString()) {
                         password.setError("Password must match to confirm password!")
                         confirmPassword.setError("Confirm Password must match to password!")
                     }
+                    // Create an error message if criteria  does not match
                 } else {
-                    fullName.setError("Full Name must be between 6 to 30 character long")
-                    userName.setError("User Name must be between 6 to 30 character long")
-                    password.setError("Password must be between 10 to 80 character long")
-                    confirmPassword.setError("Confirm Password must be between 10 to 80 character character long")
+                    fullName.setError("Full Name must be between 6 to 36 character long")
+                    userName.setError("User Name must be between 4 to 36 character long")
+                    password.setError("Password must be between 8 to 36 character long")
+                    confirmPassword.setError("Confirm Password must be between 8 to 36 character character long")
                 }
-
+                // Create an error message if e-mail the user didn't input '@'
             } else {
-                email.setError("Please Enter a Valid Email and must be between 6 to 319 character long")
+                email.setError("Please Enter a Valid Email and must be between 6 to 30 character long")
```

Comment:
```suggestion
                email.setError("Please Enter a Valid Email")
```

Reply:
Lets just have for a bad email input "Please enter a valid email" so we don't expose our max length. For the others it should be fine.